### ISSUE TITLE
Exeption in meta data editor (combined) - pressing "validation"

### ISSUE THREAD

**Issue by stefanCCS** [kitodo/kitodo-production#4021]
Version: 3.2.1-build 29.09.2020
I have made the follwing:
Created a simple structure (monograph with 3 chapters like this):
![exp-01-1](https://user-images.githubusercontent.com/61045115/95088060-bf052500-0722-11eb-85ce-f111966ba1de.png)
save and validate -> ok

Open again in meta data editor, move one page to from chater 3 to chapter 2,  like this:
![exp-01-2](https://user-images.githubusercontent.com/61045115/95088200-ed830000-0722-11eb-91b6-869bec086881.png)

Save and then pressing "validate" leads to following exception:
```
An error has occurred:
Date/time: 2020-10-05 15:48:25
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36
User IP: 192.168.102.55
Request URI: /kitodo-3.2.1-20200929/pages/metadataEditor.jsf
Ajax request: Yes
Status code: 500
Exception type: class java.lang.IllegalStateException
Exception message: Duplicate key ORDERLABEL: " - " (attempted merging values ORDERLABEL and ORDERLABEL)
Exception UUID:
Stack trace:
java.lang.IllegalStateException: Duplicate key ORDERLABEL: " - " (attempted merging values ORDERLABEL and ORDERLABEL)
	at java.base/java.util.stream.Collectors.duplicateKeyException(Collectors.java:133)
	at java.base/java.util.stream.Collectors.lambda$uniqKeysMapMerger$0(Collectors.java:155)
	at java.base/java.util.stream.ReduceOps$3ReducingSink.combine(ReduceOps.java:174)
	at java.base/java.util.stream.ReduceOps$3ReducingSink.combine(ReduceOps.java:160)
	at java.base/java.util.stream.ReduceOps$ReduceTask.onCompletion(ReduceOps.java:959)
	at java.base/java.util.concurrent.CountedCompleter.tryComplete(CountedCompleter.java:592)
	at java.base/java.util.stream.AbstractTask.compute(AbstractTask.java:328)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinTask.doInvoke(ForkJoinTask.java:408)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:736)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateParallel(ReduceOps.java:919)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at org.kitodo.validation.metadata.MetadataValidation.checkMetadataRules(MetadataValidation.java:173)
	at org.kitodo.validation.metadata.MetadataValidation.validate(MetadataValidation.java:129)
	at org.kitodo.production.services.validation.MetadataValidationService.validate(MetadataValidationService.java:182)
	at org.kitodo.production.forms.dataeditor.DataEditorForm.validate(DataEditorForm.java:320)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

```

**Comment by solth** [comment]
@oliver-stoehr shouldn't this have been fixed by #3986 ?

**Comment by oliver-stoehr** [comment]
#3986 fixed only one occurrence of this exception, #4021 should fix all of them.

**Comment by stefanCCS** [comment]
Confirmed - fixed in version 3.2.1-build 07.10.2020
